reservation/src/navigation/Navigator.tsx

```diff
@@ -22,7 +24,12 @@ export const Navigation = () => {
       <Stack.Screen
         name="HomeScreen"
         component={HomeScreen}
-        options={{title: 'Home', headerShown: true}}
+        options={{title: 'Home', headerShown: false}}
+      />
+      <Stack.Screen
+        name="LoginScreen"
+        component={LoginScreen}
+        options={{title: 'Login', headerShown: false}}
       />
```

Comment:
If you want to hide the header on all your screens or on more screens than it should be shown, you could change the option to your Stack.Navigator as screenOptions={{headerShown: false}} to avoid hide it in each screen individually